test/postsubmit-tests-with-pipeline-deployment.sh

```diff
@@ -69,6 +74,10 @@ done
 TEST_RESULTS_GCS_DIR=gs://${TEST_RESULT_BUCKET}/${PULL_BASE_SHA}/${TEST_RESULT_FOLDER}
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
 
+if [ $KFP_DEPLOYMENT != standalone ]; then
```

Comment:
nit: always try to use curly braces for bash vars
```suggestion
if [ ${KFP_DEPLOYMENT} != standalone ]; then
```